code/datums/components/shield.dm

```diff
@@ -135,10 +135,24 @@
 		return
 	shield_affect_user(user)
 
+	var/obj/item/parent_item = parent //Apply in-hand slowdowns.
+	if(!ishuman(user) || !parent_item)
+		return
+	var/mob/living/carbon/human/human_user = user
+	if(parent_item.slowdown)
+		human_user.add_movespeed_modifier(parent.type, TRUE, 0, NONE, TRUE, parent_item.slowdown)
```

Comment:
atomize

Reply:
Why am I atomizing this? It's part of the overall fix.

Reply:
Because its a balance change not a fix

Reply:
It's both really; the slowdown is 100% as it was always meant to balance the shield, and it's a bug that it doesn't work as intended.

Reply:
I dont see you correcting any errors nor can I find where the original shield PR says they are supposed to slow so atomize it so I can merge

Reply:
I dont see you correcting any errors nor can I find where the original shield PR says they are supposed to slow so atomize it so I can merge

Reply:
So, why would it have a slowdown value defined if it wasn't intended to slowdown?

You know what, I don't care, in the interests of getting this shield cancer dealt with as soon as possible I'll play along.